static/js/utils/place_utils.ts

```diff
@@ -102,6 +106,23 @@ export function getParentPlacesPromise(
     .catch(() => []);
 }
 
+/**
+ * Used to get child places (filtered by wanted place type list.
+ * Returns lists of NamedPopPlace keyed by place type.
+ */
+export function getChildPlacesPromise(
+  placeDcid: string
+): Promise<ChildPlacesByType> {
+  return axios
+    .get(`/api/place/child/${placeDcid}`)
+    .then((resp) => {
+      return resp.data;
+    })
+    .catch(() => {
+      return undefined;
```

Comment:
maybe return `{}` so that return type is the same in the then and catch?

Reply:
ah good point. was fighting lint a little here. done